oximeter/oximeter/src/types.rs

```diff
@@ -330,6 +330,23 @@ impl DatumType {
                 | DatumType::HistogramF64
         )
     }
+
+    /// Return `true` if this datum type is a histogram.
+    pub fn is_histogram(&self) -> bool {
+        matches!(
+            self,
+            DatumType::HistogramI8
+                | DatumType::HistogramU8
+                | DatumType::HistogramI16
+                | DatumType::HistogramU16
+                | DatumType::HistogramI32
+                | DatumType::HistogramU32
+                | DatumType::HistogramI64
+                | DatumType::HistogramU64
+                | DatumType::HistogramF32
+                | DatumType::HistogramF64
+        )
+    }
```

Comment:
Is this being used? Looks like we care about `is_cumulative`, but I can't see this being used in this PR

Reply:
Nope, it's not used anywhere, I'll remove it for now.